packages/genesys-spark-components/src/components/stable/gux-list/gux-list.tsx

```diff
@@ -23,7 +23,7 @@ import { trackComponent } from '@utils/tracking/usage';
  * @slot - collection of gux-list-item, gux-list-divider elements
  */
 
-const validFocusableItems = ['gux-list-item'];
+const validFocusableItems = ['gux-list-item', 'gux-avatar-list-item-beta'];
```

Comment:
I was wondering what other peoples thoughts are on this as I need to do something similar in the `text-editor` component. I need to be able to customise the styles of the `gux-list-item` but I cannot do that due to the shadowDOM. If I were to create my own custom component I would add it here like `gux-rich-style-list-item` but then wouldn't that component be available for devs to use in there own `gux-lists` which I am not sure we if we want ? 🤔  Like with this change devs could use the `gux-list` and slot `gux-avatar-list-item-beta`. Maybe something for collab time.

Reply:
Would it be worth creating a separate gux-avatar-list?

Reply:
Yeah I was going to go down that route aswell but then you will have muiltiple `gux-*-list` components (even though they would be internal) with the same functionality. Maybe thats fine 🤔  or it might be possible to centralise the `gux-list` a bit. In my current branch I turned the `gux-list-item.scss` styles into a mixin and I reference it in my `gux-rich-style-list-item` style file as it would be duplicated scss essentially.